earthly crashes when `conversion_parallelism: 0`

```
$cat ~/.earthly/config.yml 
global:
  conversion_parallelism: 0
```

```
$ cat Earthfile 
VERSION 0.7
FROM alpine:3.17
RUN echo hello
```

```
$ earthly-v0.7.4 +base
 Init 🚀
————————————————————————————————————————————————————————————————————————————————

           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)

 Build 🔧
————————————————————————————————————————————————————————————————————————————————

         alpine:3.17 | --> Load metadata alpine:3.17 linux/amd64
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xe5b38c]

goroutine 277 [running]:
github.com/earthly/earthly/util/syncutil/semutil.(*MultiSem).Acquire(0xc00088d308, {0x1702148, 0xc0003df860}, 0xc000590e48?)
	/earthly/util/syncutil/semutil/multisem.go:32 +0xac
github.com/earthly/earthly/earthfile2llb.(*waitBlock).waitStates.func1()
	/earthly/earthfile2llb/wait_block.go:282 +0x79
github.com/earthly/earthly/util/syncutil/serrgroup.(*Group).Go.func1()
	/earthly/util/syncutil/serrgroup/serrgroup.go:73 +0x67
created by github.com/earthly/earthly/util/syncutil/serrgroup.(*Group).Go
	/earthly/util/syncutil/serrgroup/serrgroup.go:70 +0xfb
```

--- a/cmd/earthly/build_cmd.go
+++ b/cmd/earthly/build_cmd.go
@@ -464,10 +464,10 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 			cacheExport = app.remoteCache
 		}
 	}
-	var parallelism semutil.Semaphore
-	if app.cfg.Global.ConversionParallelism != 0 {
-		parallelism = semutil.NewWeighted(int64(app.cfg.Global.ConversionParallelism))
+	if app.cfg.Global.ConversionParallelism <= 0 {
+		return fmt.Errorf("configuration error: \"conversion_parallelism\" must be larger than zero")
 	}
+	parallelism := semutil.NewWeighted(int64(app.cfg.Global.ConversionParallelism))
 	localRegistryAddr := ""
 	if isLocal && app.localRegistryHost != "" {
 		lrURL, err := url.Parse(app.localRegistryHost)

